packages/mint-components/src/components/sqm-portal-footer/sqm-portal-footer.feature

```diff
@@ -50,7 +50,24 @@ Feature: Portal Footer
             | has prop          |       | isn't shown |
             | doesn't have prop |       | is shown    |
 
+    Scenario Outline: Support text is shown by default
+        Given the footer <mayHaveProp> "hide-support-text" with <value>
+        Then the support text <mayBeShown>
+        Examples:
+            | mayHaveProp       | value | mayBeShown  |
+            | has prop          | true  | isn't shown |
+            | has prop          | false | is shown    |
+            | has prop          | test  | isn't shown |
+            | has prop          |       | isn't shown |
+            | doesn't have prop |       | is shown    |
+
     Scenario: Powered By Saasquatch links out to "https://saasquatch.com"
         Given the footer has the powered by SaaSquatch image
         When a user clicks on it
-        Then they will be redirected to "https://saasquatch.com" in a new page
\ No newline at end of file
+        Then they will be redirected to "https://saasquatch.com" in a new page
```

Comment:
@lillyszatmary Zoe has asked that this changes to https://www.saasquatch.com/?utm_source=app&utm_medium=user-widget&utm_campaign=referral-widget